examples/cpptest/Makefile

```diff
@@ -1,37 +1,21 @@
-PROG_NAME = cpptest
+BUILD_DIR=build
+SOURCE_DIR=.
+include $(N64_INST)/include/n64.mk
 
-ROOTDIR = $(N64_INST)
-GCCN64PREFIX = $(ROOTDIR)/bin/mips64-elf-
+cpptest.z64: MAPCMD=-Wl,-Map
```

Comment:
I don't love that this will need to be part of the Makefile pattern for any C++ project that uses n64.mk. It seems like a leaky implementation detail, but I don't have a better solution off the top of my head.